cime_config/buildnml

```diff
@@ -203,15 +203,16 @@ def buildnml(case, caseroot, compname):
         clm_usrdat_name = case.get_value("CLM_USRDAT_NAME")
         clmusr = " -clm_usr_name %s " % clm_usrdat_name
         # Write warning about initial condition data
-        if "NEON" in clm_usrdat_name and clm_force_coldstart == "off":
-            if ("_transient" in clm_nml_use_case) and (
-                re.fullmatch(r"\w\w\w\w\.transient", casename) is None
-                or clm_usrdat_name is "NEON.PRISM"
-            ):
-                logger.warning(
-                    "WARNING: Do you have appropriate initial conditions for this simulation?"
-                    + " Check that the finidat file used in the lnd_in namelist is appropriately spunup for your case"
-                )
+        if ("NEON" in clm_usrdat_name) or ("PLUMBER" in clm_usrdat_name):
+            if clm_force_coldstart == "off":
+                if ("_transient" in clm_nml_use_case) and (
+                    re.fullmatch(r"\w\w\w\w\.transient", casename) is None
+                    or clm_usrdat_name is "NEON.PRISM"
+                ):
+                    logger.warning(
+                        "WARNING: Do you have appropriate initial conditions for this simulation?"
+                        + " Check that the finidat file used in the lnd_in namelist is appropriately spunup for your case"
+                    )
```

Comment:
From looking at a different PR, I see that CLM_NML_USE_CASE has the "_transient" suffix for various compsets (Hist and SSP), so I'm retracting my comment as unnecessary.

Reply:
Just to add to the conversation, but also point out something I'm concerned about. CLM_NML_USE_CASE is better to use than the compset long-name because you might change it after you create the case. And there is a standard for having transient use-cases end in _transient, and build-namelist checks for it -- so it is the safe thing to do.

**_The thing I'm concerned about:_**
But, this also had me look at the reg-ex above, and it looks wrong to me. It's checking for 4 word characters a "." and then the transient ending. And that looks to me like it wouldn't match: 2018-PD_transient. And it should probably be setup to just check for the _transient ending rather than add additional requirements. So does this actually work in practice?

Reply:
Hmm. The warning `WARNING: Do you have appropriate initial conditions for this simulation?` certainly shows up regularly in running transient cases, which means that this does get through the if statements. I think the NEON cases specifically are set up to be named `<4-letter-neonsite-name>.transient`-- so maybe this wouldn't work for non-neon cases, but works for neon cases? This might need to change for PLUMBER case names (eg, `AR-SLu`), though?